server/util/clickhouse/clickhouse.go

```diff
@@ -179,7 +180,7 @@ func buildExecution(in *repb.StoredExecution, inv *sipb.StoredInvocation) *schem
 		Command:                            inv.GetCommand(),
 		Success:                            inv.GetSuccess(),
 		InvocationStatus:                   inv.GetInvocationStatus(),
-		Tags:                               []string{},
+		Tags:                               invocation_format.ConvertDbTagsToOlap(inv.GetTags()),
```

Comment:
nit: not this PR, but the function name should be ConvertDBTagsToOLAP for go conventions.